keras_retinanet/bin/debug.py

```diff
@@ -94,6 +94,7 @@ def create_generator(args):
         generator = PascalVocGenerator(
             args.pascal_path,
             args.pascal_set,
+            args.image_extension,
```

Comment:
Without the argument name this is assigned to `classes`.

Reply:
I don't get you.

Reply:
The arguments for the `PascalVocGenerator` are:
```python
    def __init__(
        self,
        data_dir,
        set_name,
        classes=voc_classes,
        image_extension='.jpg',
        skip_truncated=False,
        skip_difficult=False,
        **kwargs
    ):
```

So the third argument would be `classes`, where you pass `args.image_extension`.